zenoh/src/session.rs

```diff
@@ -11,14 +11,17 @@
 // Contributors:
```

Comment:
I didn't see the renaming of `declare_expr` into `declare_keyexpr`.

Reply:
because I borked my grep in the API harmonisation discussion xD